### PR TITLE
Issue #662

### PR DESCRIPTION
**Pull request by joergboe**

### CODE REVIEW

**Comment by ddebrunner** [comment]
Can one of the admins verify this patch?

File: com.ibm.streamsx.topology/bin/spl-python-extract.py [hunk 1]
```diff
@@ -181,34 +181,31 @@ def setupInfoXml(languageList):
...
-                Sample info xml:
-                """)
-            print(_INFO_XML_TEMPLATE)
+            errstr = """"ERROR: Message set for the "TopologySplpyResource" is missing. Correct the resource section in info.xml file.
+
+                Sample info xml:\n""" + _INFO_XML_TEMPLATE
+            sys.exit(errstr)
     except FileNotFoundError as e:
-        print("File info.xml not found. Create info.xml from template")
+        print("WARNING: File info.xml not found. Create info.xml from template")
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/bin/spl-python-extract.py]: Should "create" be "creating" here? As it is the utility that it creating `info.xml`, not an instruction for the user to create it?

Comment (joergboe) [inline com.ibm.streamsx.topology/bin/spl-python-extract.py]: Yes The utility creates the info.xml from template

**Comment by joergboe** [comment]
Corrected

**Comment by ddebrunner** [comment]
test now

**Pull request merged**